frontend/public/style/_overrides.scss

```diff
@@ -466,3 +466,5 @@ button.pf-c-dropdown__menu-item.pf-m-disabled {
   // enables tooltips for disabled menu items
   pointer-events: auto;
 }
+
+.pf-c-drawer__content { --pf-c-drawer__content--ZIndex: auto; }
```

Comment:
Sorry.  On second thought, I think this should go in `_notification-drawer.scss` so all the drawer scss is in the same file.

Reply:
@rhamilto, moved to `_notification-drawer.scss` and used multiple lines :-)